Formula/gpsbabel.rb

```diff
@@ -12,15 +12,20 @@ class Gpsbabel < Formula
     sha256 "e982a298816049c9094762699799f238cfc8d7804cf5d72f6816ebd0e8aa414e" => :sierra
   end
 
+  depends_on "pkg-config" => :build
   depends_on "libusb"
   depends_on "qt"
 
   def install
     ENV.cxx11
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make", "install"
+    # on macOS force use of homebrew libusb-1.0 instead of included version.
+    # on linux correct include path for homebrew libusb-1.0.
```

Comment:
Can you link one of the issues from upstream about using system libusb on Mac here?

Reply:
Wouldn't this fall under https://docs.brew.sh/Formula-Cookbook#inreplace

> `inreplace` should be used instead of patches when patching something that will never be accepted upstream, e.g. making the software’s build system respect Homebrew’s installation hierarchy.

The next sentence seems to imply that a patch/PR/issue should be made only if it doesn't build on macOS:

> If it’s something that affects both Homebrew and MacPorts (i.e. macOS specific) it should be turned into an upstream submitted patch instead.

Reply:
If it's a couple of lines we can just use inreplace. If it's larger we can't really avoid patches.

You should always submit pull requests for anything that's broken because it's the right thing to do 😁. I think the documentation here is incompleted compared to our intent to carry as little patches as possible.

Reply:
As one of the upstream maintainers I can tell you we would reject the patch corresponding to the inreplace lines because they are too specific.  It is often unexpectedly difficult to simultaneously meet the needs of all the distributions, including our own packaging for windows and macOS available at gpsbabel.org, and not overburden the code base.  Almost all distributions currently use specific targeted patches to bridge the gap between what is supplied upstream and their desires.

What would be useful is if someone who is familiar with homebrew's policies and desires could open an issue on https://github.com/gpsbabel/gpsbabel.git stating your desire and motivation, e.g. "It is homebrews policy to use ... because ... .  Can you please provide a method to support this?"  Having a homebrew representative open the request will make it possible for the upstream maintainers to ask further questions.  I warn you that I don't think a resolution will be forthcoming in the short term, we have been contemplating how to reduce the number of build systems we support.

Reply:
Done: https://github.com/gpsbabel/gpsbabel/issues/608

Reply:
@tsteven4 you mentioned "upstream is unlikely to add customization options for library location, similar recent requests have not been accepted.", could you link one of the similar requests here?

Reply:
Here is a PR that attempted to add configurability for shapelib: https://github.com/gpsbabel/gpsbabel/pull/387
Here is a new PR that adds configurability to the qmake flow for zlib/shapelib/libusb: https://github.com/gpsbabel/gpsbabel/pull/611

BTW homebrew doesn't have pkg-config support for shapelib.

Reply:
The lack of pkg-config for shapelib is due to https://github.com/OSGeo/shapelib/issues/13, although presumably this could be added in the shapelib formula, or shapelib formula could use configure instead of cmake.